modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/CheckboxBase.tsx

```diff
@@ -132,8 +132,9 @@ export default function CheckboxBase({
 					...(otherProps.tip && {
 						'aria-describedby': `${otherProps.id ?? name}_fieldHelp`,
 					}),
-					...(otherProps.errorMessage && {
+					...(otherProps.displayErrors && otherProps.errorMessage && !otherProps.valid && {
```

Comment:
If you guys point out to me that this approach is correct, I will need to condition the `aria-errormessage` using `displayErrors` in the other field types.

Because the modification made in `reactFieldBase` is general for all fields.